cmd/registry/cmd/upload/manifest.go

```diff
@@ -85,7 +85,7 @@ func manifestCommand(ctx context.Context) *cobra.Command {
 			}
 
 			artifact := &rpc.Artifact{
-				Name:     "projects/" + projectID + "/locations/global/artifacts/" + manifest.Name,
+				Name:     "projects/" + projectID + "/locations/global/artifacts/" + manifest.Id,
```

Comment:
Since we are making a change here, can we use `manifest.GetId()`? That seems like the right approach to get the field, the way it is done in `style_guide.go`